release-notes/release-notes54.md

```diff
@@ -23,7 +23,22 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.4 release.
 
-### 
+### Updates and additions to the Table plugin
+
+The {{site.productname}} 5.4 release includes the following improvements for the table (`table`) plugin:
+
+- Adds: commands, APIs, toolbar buttons, icons, and menu items for; cut, copy, and paste columns.
```

Comment:
menu items and toolbar buttons shouldn't be listed here.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1563/commits/60291ef6cfd53d51c46a56445b277f5fd9bd643c